docs/tutorials/uploading-photo-to-a-product.md

```diff
@@ -0,0 +1,119 @@
+# Uploading Images to the Open Food Facts API
+<!--Add a brief introduction of what the tutorial does -->
+
+This basic tutorial shows you can upload image of a product to the Open Food Facts API.
+
+## Points to consider before uploading photos
+
+### Image license
+
+Products images MUST be under the [Creative Commons Attribution ShareAlike licence](https://creativecommons.org/licenses/by-sa/3.0/deed.en)(https://creativecommons.org/licenses/by-sa/3.0/deed.en).
+
+That means you should either upload: 
+* photos that are your own work
+* photos taken by your users, with their consent for this license (should be part of your service terms)
+* photos already under this license or a more liberal one
```

Comment:
I would remove the phrasing "or a more liberal one" because it can be easely misinterpreted. Let me know your view on this one though

Reply:
I am not entirely sure the license context, @alexgarel  will be in the best position to answer.

Reply:
I agree. Let's be more precise to remove ambiguity.

```suggestion
* photos already under this license or a compatible license ([cc-by](https://creativecommons.org/licenses/by/4.0/), [cc-0](https://creativecommons.org/share-your-work/public-domain/cc0/) or public domain)
```